lib/tether/ops_linux.go

```diff
@@ -872,6 +891,13 @@ func (t *BaseOperations) Fork() error {
 }
 
 func (t *BaseOperations) Setup(config Config) error {
+	if err := createBindSrcTarget(filesForMinOSLinux); err != nil {
+		return err
+	}
+
+	Sys.Hosts = etcconf.NewHosts(hostsPathBindSrc)
```

Comment:
`Setup` is called per tether instance, while Sys is package global. You should put this in an init() function.